### PR TITLE
add btc single sig address and sign demo

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

File: gowrapper/main.go [hunk 1]
```diff
@@ -187,6 +190,65 @@ func (device *jsDevice) AsyncChannelHashVerify(done func(*jsError), ok bool) {
 	}()
 }
 
+func (device *jsDevice) AsyncBTCAddressSimple(
+	done func(string, *jsError),
+	coin messages.BTCCoin,
+	keypath []uint32,
+	typ messages.BTCScriptConfig_SimpleType,
```
Comment (Tomasvrba) [inline gowrapper/main.go]: typ -> type

File: gowrapper/main.go [hunk 2]
```diff
@@ -187,6 +190,65 @@ func (device *jsDevice) AsyncChannelHashVerify(done func(*jsError), ok bool) {
...
+	display bool) {
+	go func() {
+		address, err := device.device.BTCAddress(
+			coin,
+			keypath,
+			firmware.NewBTCScriptConfigSimple(typ),
```
Comment (Tomasvrba) [inline gowrapper/main.go]: same

File: gowrapper/main.go [hunk 3]
```diff
@@ -187,6 +190,65 @@ func (device *jsDevice) AsyncChannelHashVerify(done func(*jsError), ok bool) {
...
+	if err != nil {
+		return err
+	}
+	return json.Unmarshal(jsonBytes, out)
+}
+
+func (device *jsDevice) AsyncBTCSignSimple(
+	done func([][]byte, *jsError),
+	coin messages.BTCCoin,
+	typ messages.BTCScriptConfig_SimpleType,
```
Comment (Tomasvrba) [inline gowrapper/main.go]: same

File: demo/demo.js [hunk 4]
```diff
@@ -65,19 +65,87 @@ export async function demo() {
...
+        }
+    });
+
     document.getElementById("ethPub").addEventListener("click", async () => {
         const pub = display => firmware.js.AsyncETHPub(
             firmwareAPI.messages.ETHCoin.ETH,
             [44 + HARDENED, 60 + HARDENED, 0 + HARDENED, 0, 0],
             firmwareAPI.messages.ETHPubRequest_OutputType.ADDRESS,
             display,
-            new Uint8Array(),
```
Comment (Tomasvrba) [inline demo/demo.js]: The `contractAddress []byte` is still required by AsyncETHPub so this breaks the display eth address button. It throws an error now

Comment (benma) [reply demo/demo.js]: Reverted.

**Comment by Tomasvrba** [comment]
Before you merge, I added a CHANGELOG, would be nice to include the additions there for each relevant PR.

Also starting this for the app would be great.
I'd suggest adding a PR template with a checklist for the person who opens the PR. 

I don't have the permissions to add templates for PRs though.

**Pull request merged**